docs/platform_ccl.rst

```diff
@@ -0,0 +1,297 @@
+``cycle_time`` is the value of the clock cycle time.
+
+Rest of the entries from line 6 to 14 specify various buffer times to be inserted in various operations due to control electronics setup. For example, ``mw_mw_buffer`` can be used to specify time to be inserted between a microwave operation followed by another microwave operation.
+
+.. note:: 
+	It is important to use same units to specify time in the configuraton
+	file. OpenQL uses ``cycle_time`` to convert all the times to cycles. 
+	These cycles are then used internally to schedule various operations.
+
+
+``resources`` section is used to specify/configure various resources
+available in the platform as discussed below. CC-Light architecture 
+assumes the following connections:
+
+.. _table_ccl_connections:
+
+.. table::
+	:align: center
+
+	=====================    =============   =============      =================== 
+	   Device Name           DIO connector   Target qubits        Operation Type    
+	=====================    =============   =============      =================== 
+	 UHFQC-0                   DIO1          0, 2, 3, 5, 6          measurement       
+	 UHFQC-1                   DIO2             1, 4                measurement       
+	 AWG-8 0, channel 0~6      DIO3             0~6                    flux              
+	 AWG-8 1, channel 0        DIO4             0,1                  microwave         
+	 AWG-8 1, channel 1        DIO4             5,6                  microwave         
+	 AWG-8 2, channel 0        DIO5            2,3,4                 microwave         
+	 VSM                        --              0~6              microwave masking 
+	=====================    =============   =============      =================== 
+
+Qubits available in the pltform are specified in ``qubits`` section, as shown below. For CC-Light only ``count`` needs to be specified which indiates the number of available qubits.
+
+.. code-block::
+   :linenos:
+
+	"qubits":
+	{
+	    "count": 7
+	},
+
+An instruction may require a wave form generator resource. Waveform generators and their constraints are specified in ``qwgs`` section, as shown below.
+
+.. code-block::
+   :linenos:
+
+	"qwgs" :
+	{
+	  "count": 3,
+	  "connection_map":
+	  {
+	    "0" : [0, 1],
+	    "1" : [2, 3, 4],
+	    "2" : [5, 6]
+	  }
+	},
+
+The number of these waveform generators is specified by the ``count`` field. In the ``connection_map`` it is specified which wafeform generator is connected to which qubits. For instance, Line 6 specifies that ``qwg 0`` is connected to qubits 0 and 1. This is based on ``AWG-8 1, channel 0`` entry in 4th row in Table :numref:`table_ccl_connections` This information is utilized by the scheduler to perform resource-constraint aware scheduling of instructions.
+
+.. note::
+	By providing an empty list for a qwg will result in not applying any qwg constraint during scheduling.
+
+Available measurement/readout units are specified in ``meas_units`` section, as shown below.
```

Comment:
Again, no information on what constraints this information will translate into.
This should be added, so users understand how these influence the scheduling (and/or anything else) of their programs